test-utils/testlib/src/user/runtime_user.rs

```diff
@@ -211,6 +212,14 @@ impl User for RuntimeUser {
             .map_err(|err| err.to_string())
     }
 
+    fn view_contract_code(&self, account_id: &AccountId) -> Result<ContractCode, String> {
+        let state_update = self.client.read().expect(POISONED_LOCK_ERR).get_state_update();
+        self.trie_viewer
+            .view_contract_code(&state_update, account_id)
+            .map(|account| account.into())
```

Comment:
Why is this `.into` needed? Also, it seems that it is not an account there, but a contract code.